torchvision/prototype/features/_bounding_box.py

```diff
@@ -19,6 +19,13 @@ class BoundingBox(_Feature):
     format: BoundingBoxFormat
     image_size: Tuple[int, int]
 
+    @classmethod
+    def _wrap(cls, tensor: torch.Tensor, *, format: BoundingBoxFormat, image_size: Tuple[int, int]) -> BoundingBox:
```

Comment:
Nit: Do we need another private method across all classes? Why not add this directly to `wrap_like` and call that method when necessary (for example from the `__new__`).

Reply:
> Why not add this directly to `wrap_like` and call that method when necessary (for example from the `__new__`).

As it stand, `wrap_like` needs a reference instance of that class. Since we don't have that inside the constructor, I've factored out the common parts into `_wrap`. 

If you are worried about the performance implications, they are minimal. Using `features.Image._wrap(output, color_space=input.color_space)` yields

```py
Overhead per sample:  18.8 µs
Estimated overhead per training: 0.47 h
```

This is the best we can do, i.e. we pass everything explicitly. The difference is roughly 100 ns per call or roughly 3 minutes for the whole training. If performance was the only concern, I would keep both methods for convenience and parity between the classes.